pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/ConsumerStats.java

```diff
@@ -69,8 +69,8 @@ public interface ConsumerStats {
     /** Flag to verify if consumer is blocked due to reaching threshold of unacked messages. */
     boolean isBlockedConsumerOnUnackedMsgs();
 
-    /** The read position of the cursor when the consumer joining. */
-    String getReadPositionWhenJoining();
+    /** Last sent positions per sticky key of the cursor when the consumer joining. */
```

Comment:
This is a breaking API change, please do not do it

you can add the new method but we should keep the old value as well, otherwise we cannot cherry-pick to old branches